src/libraries/native-binplace.proj

```diff
@@ -6,6 +6,7 @@
     <BinPlaceRuntime>false</BinPlaceRuntime>
     <BinPlaceNative>true</BinPlaceNative>
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <IsSourceProject>false</IsSourceProject>
```

Comment:
This property is expected to be set in a Directory.Build.props file. Can you please move it there? Reason is that some properties in the root Directory.Build.props file condition on it and when setting it in the project file, it's too late.